MHCollapsibleViewManager.m

```diff
@@ -0,0 +1,228 @@
+//
+//  MHCollapsibleViewManager.m
+//  This is a manager of a uiview and view controller
+//
+//  Created by Lizzy Randall on 5/6/15.
+//  Copyright (c) 2015 Lizzy Randall. All rights reserved.
+//
+
+#import "MHCollapsibleViewManager.h"
+#import "MHFilterLabel.h"
+
+@implementation MHCollapsibleViewManager
+
+//@synthesize tableView;
+//@synthesize viewController;
+@synthesize rowAnimation;
+
+@synthesize data;
+@synthesize cellStyles;
+@synthesize headerTitle;
+
+@synthesize expanded;
+@synthesize numOfRows;
+
+//Initialize with animation, title and tableView
+- (id)initalizeManager:(UITableViewRowAnimation)animation title:(NSString*) title tableView:(UITableView*)tableView
+{
+    self.headerTitle = title;
+    self.rowAnimation = animation;
+    //initialized first and then can be overwritten by controller
+    //tableView is needed to create cells to store types
+    [self setCellProperties:nil header:nil clickedCell:nil normalCell:nil tableView:tableView];
+    return self;
+}
+
+//called after Initializing Manager
+- (void) initializeData:(NSMutableArray*)dataFromController
+{
+    self.data = [[NSMutableArray alloc] init];
+    for(int i=0; i < [dataFromController count] ; i++)
+    {
+        [data addObject:[MHFilterLabel constructLabel:[dataFromController objectAtIndex:i] checked:false]];
+    }
+    self.numOfRows = (NSInteger)[data count];
+}
+
+//set the cell provided with same settings as corresponding array type
+- (UITableViewCell*)setCellSettings:(UITableViewCell*)cell type:(NSString*)type
+{
+    UITableViewCell *tempCell = nil;
+    int index = 3; //normall cell
+    if([type isEqual: @"ClickedHeader"])
```

Comment:
this is just personal preference but we have been using trailing brackets in all the objc code bases so far. I'm not sure what is standard in the java or ruby code bases. We should probably talk about that and make it consistent with what the rest of the team does.
if() {
...
}